### ISSUE TITLE
Console Error on Events configuration

### ISSUE THREAD

**Issue by szczukot** [domoticz/domoticz#3724]
The latest beta 4.11474

Go to Settings->Events

Errors on console:
blockly_compressed.js:1191 Uncaught (in promise) DOMException: play() failed because the user didn't interact with the document first. https://goo.gl/xX8pDD
Blockly.WorkspaceAudio.preload @ blockly_compressed.js:1191
e @ blockly_compressed.js:1690
h @ blockly_compressed.js:1699

Maybe upgrade blockly to the latest version will resolve this problem ?

**Comment by szczukot** [comment]
I tested locally: with the latest blockly js files, this error not occurs.

**Comment by gizmocuz** [comment]
I don't have this error here, but I will update blockly for you.
What browser are you using ? Do you experience this with Chrome as well ?

**Comment by szczukot** [comment]
Chrom 78.0.3904.97 on Win10

**Comment by waltervl** [comment]
Still an issue? Could not find a commit updating Blockly around this time.

**Comment by szczukot** [comment]
stilll exists (the latest chrome and domoticz)
http://192.168.1.2:8080/#/Events
![image](https://user-images.githubusercontent.com/10205661/156040916-0b1abc10-ee4f-4cc7-b4f7-6ea57ea64d3f.png)

pls reopen.

**Comment by waltervl** [comment]
I can confirm the same error in Chrome. 
In Firefox it is displayed as
`Uncaught (in promise) DOMException: The fetching process for the media resource was aborted by the user agent at the user's request.`
@szczukot Can you make a PR for this to update the latest blockly sources?

**Comment by szczukot** [comment]
@waltervl https://github.com/domoticz/domoticz/issues/1957#issuecomment-348464121

**Comment by waltervl** [comment]
That issue 1957 is about the math module that  cannot be added. I thought to read you had already fixed it with updated blockly code?

**Comment by szczukot** [comment]
I didn't get "permission" to update the blockly

_"Gizmocuz, can you upload the latest version?
The current one is 1.5 years old and is very old.


gizmocuz commented
NO, and this can not be added as we need to support this as well.
As you might already know, if you want to use a more advanced script, you can use dzVents or Pytjon"_

**Comment by waltervl** [comment]
Please test it in your test environment (I do not use Blockly) to use the latest Blockly.js sources. And then make a PR if successful. 

It looks like you had a communication error. Gizmocuz (and the issue) was about the blockly math library to be added.

**Comment by gizmocuz** [comment]
Yes indeed.... You can add the math library but you need to add support for this as well. Maybe two separate PR's, one to update blockly and the other to add math support ?
